generators/kubernetes/templates/keycloak/keycloak-postgresql.yml.ejs

```diff
@@ -28,9 +28,12 @@ spec:
   resources:
     requests:
       storage: 2Gi
-  <%_ if (kubernetesStorageClassName) { _%>
+  <%_ if (kubernetesStorageClassName && !minikube) { _%>
   storageClassName: <%= kubernetesStorageClassName %>
   <%_ } _%>
+  <%_ if (kubernetesStorageClassName && minikube) { _%>
+  storageClassName: standard
+  <%_ } _%>
```

Comment:
`kubernetesStorageClassName` value can be set as `standard` in the logic rather than adding redundant condition here.